app/actions/__tests__/searchActions.spec.js

```diff
@@ -1,8 +1,8 @@
 import { expect } from 'chai';
 import simple from 'simple-mock';
 
-import searchActions from 'actions/searchActions';
-import apiUtils from 'utils/APIUtils';
+import { searchResources } from 'actions/searchActions';
+import * as apiUtils from 'utils/APIUtils';
```

Comment:
Why in tests we have to import modules using `* as`? 

I guess this line ```import { searchResources } from 'actions/searchActions';``` is ok in the tests but it wouldn't be ok out of them?